src/cmd/build.js

```diff
@@ -242,6 +242,7 @@ export default async function build(
           throw error;
         });
       },
+      reloadDelay: new Number(1000),
```

Comment:
Why are you using the `Number` constructor here?

The constructors of [primitive values](https://developer.mozilla.org/en-US/docs/Glossary/Primitive) are rarely used because it is not necessary. The code (in this whole PR) would be more idiomatic if you use the primitive value directly, without wrapping it as an object.